src/ifd-nfc.c

```diff
@@ -444,8 +519,12 @@ IFDHGetCapabilities(DWORD Lun, DWORD Tag, PDWORD Length, PUCHAR Value)
       *Value  = 1;
       *Length = 1;
       break;
+#if defined(HAVE_DECL_TAG_IFD_STOP_POLLING_THREAD) && HAVE_DECL_TAG_IFD_POLLING_THREAD
```

Comment:
You should only use:
```
#ifdef HAVE_DECL_TAG_IFD_STOP_POLLING_THREAD
```
If ``HAVE_DECL_TAG_IFD_STOP_POLLING_THREAD`` is defined but has value 0 then your code is wrong.